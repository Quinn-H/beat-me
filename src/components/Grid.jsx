import React from 'react'

import Track from './Track'

const Grid = (props) => {
  const tracks = props.tracks.map((track, index) => <Track key={index} track={track} />)
  return (
    <div className="container">
      <table className="table">
        <tbody>
          {tracks}
        </tbody>
      </table>
    </div>
  )
}

export default Grid

// <tr className="track">
//   <th className="row">Synth</th>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
// </tr>
// <tr className="track">
//   <th className="row">Kick</th>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
//   <td><button type="button" className="btn btn-outline-primary"></button></td>
// </tr>
