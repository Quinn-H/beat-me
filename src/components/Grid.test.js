/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Grid from './Grid'

test('Grid renders correctly', () => {
  const tracks = []
  const tree = renderer.create(
    <Grid tracks={tracks} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
