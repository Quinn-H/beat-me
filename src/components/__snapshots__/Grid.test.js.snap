exports[`test Grid renders correctly 1`] = `
<div
  className="container">
  <table
    className="table">
    <tbody />
  </table>
</div>
`;
